README.md

```diff
@@ -2,27 +2,28 @@
-| [github__daily_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__daily_metrics)     | Each record represents a single day, enriched with metrics about PRs and issues that were created and closed during that period.                              |
-| [github__weekly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__weekly_metrics)    | Each record represents a single week, enriched with metrics about PRs and issues that were created and closed during that period.                             |
-| [github__monthly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__monthly_metrics)   | Each record represents a single month, enriched with metrics about PRs and issues that were created and closed during that period.                            |
-| [github__quarterly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__quarterly_metrics) | Each record represents a single quarter, enriched with metrics about PRs and issues that were created and closed during that period.                          |
+| [github__pull_requests](https://dbt-github.netlify.app/#!/model/model.github.github__pull_requests)     | Each record represents a GitHub pull request, enriched with data about its repository and reviewers, as well as the durations between review requests, merges, and reviews. |
+| [github__daily_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__daily_metrics)     | Each record represents a single day, enriched with metrics about PRs and issues that were created and closed that day.                              |
+| [github__weekly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__weekly_metrics)    | Each record represents a single week, enriched with metrics about the PRs and issues that were created and closed that week.                             |
+| [github__monthly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__monthly_metrics)   | Each record represents a single month, enriched with metrics about PRs and issues that were created and closed that month.                            |
+| [github__quarterly_metrics](https://dbt-github.netlify.app/#!/model/model.github.github__quarterly_metrics) | Each record represents a single quarter, enriched with metrics about PRs and issues that were created and closed that quarter.                          |
 
-# 🤔 Who is the target user of this dbt package?
-This package is intended to be leveraged by individuals who want to take advantage of analysis ready models to fuel their reporting needs. The package outputs reporting tables that may be easily linked directly to your BI tool of choice for quick time to value with your GitHub data.
+# 🤔 Why should I use this dbt package?
```

Comment:
Are there users who shouldn’t use this package? Why not?

Reply:
Hmmm I don't know if we should highlight if people shouldn't. Can we instead presume that people will infer that the package isn't for them if they don't fit the "why should I use this package"?

Reply:
Yeah, that's fair. I was wondering if the package works best in specific circumstances - if there are data size limits, in certain use cases, etc. If we can be specific about that, we should.